abuild.in

```diff
@@ -1803,6 +1829,50 @@ openrc() {
 	default_openrc
 }
 
+default_bashcomp() {
+	depends=""
+	pkgdesc="Bash completions for $pkgname"
+	install_if="$pkgname=$pkgver-r$pkgrel bash-completion"
+
+	cd "$pkgdir" || return 0
+	for i in $(find usr/share/bash-completion/completions etc/bash_completion.d -type f 2>/dev/null); do
```

Comment:
It may be possible to use `find(1)/-exec` here, if you `mkdir` outside of the loop.

Reply:
i'd rather keep the loop, if i try to `mkdir` from outside then i will just run into the issue of how to test if there are completion files